### ISSUE TITLE
The pt-BR translation is broken in the pop-up

### ISSUE THREAD

**Issue by GoogleCodeExporter** [max99x/pagemon-chrome-ext#133]
```
What steps will reproduce the problem?
- Click on the icon of Page Monitor using the Portuguese translation
- Note that a sentence is broken.
- Is just appearing "alterações" instead of "Ver alterações".

What is the expected output? What do you see instead?
It should display the phrase "Ver alterações"

What version of the Chrome are you using? On what operating system?
- Google Chrome 12
- Windows 7

Please provide any additional information below.
This translation has been implemented on the ticket 124 
(http://code.google.com/p/pagemon-chrome-ext/issues/detail?id=124).
```

Original issue reported on code.google.com by `davidech...@gmail.com` on 20 Jun 2011 at 11:39

**Comment by GoogleCodeExporter** [comment]
```
A picture with the bug.
```

Original comment by `davidech...@gmail.com` on 20 Jun 2011 at 4:29

Attachments:
* [bug-pagemonitor.png](https://storage.googleapis.com/google-code-attachments/pagemon-chrome-ext/issue-133/comment-1/bug-pagemonitor.png)

**Comment by GoogleCodeExporter** [comment]
```
Ah, strange. I thought I checked that it fit. Is it possible to convert any of 
these three messages to something shorter that could fit in the popup?
```

Original comment by `max99x` on 20 Jun 2011 at 7:05
* Changed state: **Accepted**
* Added labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
You can change the "marcar como visitado" to "marcar visitado".
```

Original comment by `davidech...@gmail.com` on 20 Jun 2011 at 7:17

**Comment by GoogleCodeExporter** [comment]
```
That helps, but unfortunately it's still not enough to fit the whole thing. 
Some other localizations that had trouble fitting the labels changed the 
stop_monitoring message to something like Remove or Delete. Could that be done 
here as well?
```

Original comment by `max99x` on 20 Jun 2011 at 7:22

**Comment by GoogleCodeExporter** [comment]
```
My fear is that it can be confusing to the user, but you can change the "parar 
monitoramento" to "remover".
```

Original comment by `davidech...@gmail.com` on 20 Jun 2011 at 7:26

**Comment by GoogleCodeExporter** [comment]
```
Thanks. Fixed in revision 208 and pushed in version 3.2.5. The confusion might 
be an issue, but I'm counting on the fact that one of the choices is "mark 
visited" - that should make it clear that removing in this case refers to the 
whole page rather than the notifications. If you come up with a better 
alternative of an acceptable length, I'll be happy to use it in a later version.
```

Original comment by `max99x` on 20 Jun 2011 at 7:36
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Ok, max99x and thank you again. 
If I think of something that can help you, I create a new ticket.
```

Original comment by `davidech...@gmail.com` on 20 Jun 2011 at 7:57
